code/game/objects/random/loot.dm

```diff
@@ -335,30 +335,40 @@
 		/obj/item/storage/pill_bottle/mortaphenyl = 3,
 		/obj/item/storage/pill_bottle/happy = 2,
 		/obj/item/storage/pill_bottle/zoom = 2,
+		/obj/item/reagent_containers/pill/cocaine = 3,
+		/obj/item/reagent_containers/pill/heroin = 2,
+		/obj/item/reagent_containers/hypospray/autoinjector/stimpack = 2,
+		/obj/item/reagent_containers/hypospray/autoinjector/berserk = 1,
+		/obj/item/reagent_containers/hypospray/autoinjector/impedrezene = 0.4,
+		/obj/item/reagent_containers/hypospray/autoinjector/night_juice = 0.3,
+		/obj/item/storage/box/syndie_kit/syringe_gun = 0.5,
+		/obj/item/reagent_containers/syringe/drugs = 1,
+		/obj/item/reagent_containers/inhaler/space_drugs = 2,
+		/obj/item/grenade/spawnergrenade/manhacks = 0.2,
```

Comment:
I think this is too silly to be a random maint spawn

Reply:
Disagree, especially with such a low probability chance

Reply:
I think the manhacks one is fine, it's very low and honestly it isn't that good. Would you like to see it's probability lowered even further? 

I tested this by loading a server about 8 times and checked each spawn, I didn't get a manhack a single time.

Reply:
thinking about it again
pretty sure the evisecrator/manhack thingies are antag only and crew does not see them in normal canon gameplay
idk

Reply:
Removed, I'll add in something else in the future.